pubsub/google/cloud/pubsub/topic.py

```diff
@@ -252,7 +250,7 @@ def publish(self, message, client=None, **attrs):
         api = client.publisher_api
 
         self._timestamp_message(attrs)
-        message_b = base64.b64encode(message).decode('ascii')
+        message_b = message.decode('ascii')
```

Comment:
Actually, removing this fails 6 tests. I could change the `b''` messages to just strings but I'm not sure that's the right way to go?